data-browser/src/components/Button.tsx

```diff
@@ -1,29 +1,31 @@
 import React from 'react';
 import styled from 'styled-components';
+import { Spinner } from './Spinner';
 
 interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
-  children: React.ReactNode;
+  /** Description of the button, required if the button only has an icon */
+  name?: string;
   /** Renders the button less clicky */
   subtle?: boolean;
   /** If it's just an icon */
   icon?: boolean;
-  /** If it needs no margins */
-  noMargins?: boolean;
   /** Minimal styling */
   clean?: boolean;
   /** Shows loading text + a spinner */
   loading?: string;
+  /** Add a bottom margin */
+  gutter?: boolean;
   onClick: (...a: unknown[]) => unknown;
 }
 
-export function Button({
+export const Button: React.FC<React.PropsWithChildren<ButtonProps>> = ({
```

Comment:
Any specific reason to migrate `function` components to `const` declarations?  I personally prefer `function` as it is slighly less verbose syntax.